py/tests/dict.py

```diff
@@ -33,4 +33,15 @@
 assert a.__contains__('hello')
 assert not a.__contains__('world')
 
+doc="__eq__, __ne__"
+a = {'a': 'b'}
+assert a.__eq__(3) != True
+assert a.__ne__(3) != False
+assert a.__ne__(3) != True
+assert a.__ne__(3) != False # This part should be changed in comparison with NotImplemented
```

Comment:
Can you remove this comment please?  It is in the wrong place (it should be on the line above) and it refers to a bug which is fixed (or will be when this is merged) so will make no sense in the future.

The best way of making that sort of comment is to add a github comment to the review.

Reply:
Removed it. Would you like to review it again?